test/ucp/storetest/shared.go

```diff
@@ -81,13 +86,36 @@ var Data2 = map[string]any{
 		"resource": "2",
 	},
 }
-
-var NestedData1 = map[string]any{
+var Data3 = map[string]any{
 	"value": "3",
 	"properties": map[string]any{
 		"resource": "3",
 	},
 }
+var NestedData1 = map[string]any{
```

Comment:
any reason why these variables start with capital case?

Reply:
These are exported from the package, because they are used by other tests. Everything in `storetest` is designed to be common between the tests for all of our implementations.